src/ziggurat/messaging/consumer.clj

```diff
@@ -95,7 +95,7 @@
       (start-subscriber* (lch/open connection)
                          (get-in-config [:jobs :instant :prefetch-count])
                          (prefixed-queue-name topic-entity (get-in-config [:rabbit-mq :instant :queue-name]))
-                         (mpr/mapper-func mapper-fn channels)
+                         (mpr/mapper-func mapper-fn channels {})
```

Comment:
If we are adding trace information to the rabbitmq headers, why are we not passing those to the mapper-func?

Reply:
it only needs to be added when the message is put in rabbit the very first time. from there on it gets carried forward. so it is not needed in the subscriber. the producer will take care of it